experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/edit_branches.html

```diff
@@ -0,0 +1,190 @@
+          <div class="col">
+            <div class="form-check">
+              {{ form.warn_feature_schema }}
+              <label class="form-check-label" for="id_warn_feature_schema">Warn only on feature schema validation failure</label>
+            </div>
+          </div>
+        </div>
+        <div class="row mb-3">
+          <div class="col">
+            <div class="form-check">
+              {{ form.equal_branch_ratio }}
+              <label class="form-check-label" for="id_equal_branch_ratio">Users should be split evenly between all branches</label>
+            </div>
+          </div>
+        </div>
+        <div class="row mb-3">
+          <div class="col">
+            <div class="form-check">
+              {{ form.prevent_pref_conflicts }}
+              <label class="form-check-label" for="id_prevent_pref_conflicts">
+                Prevent enrollment if users have changed any prefs set by this experiment
+              </label>
+            </div>
+          </div>
+        </div>
+        <div id="branches">
+          {{ form.branches.management_form }}
+          {% for branch_form in form.branches %}
+            <div class="card bg-body-tertiary mb-3">
+              <div class="card-body  position-relative">
+                <div class="form-group">
+                  {{ branch_form.id }}
+                  <div class="row">
+                    <div class="col-3">
+                      <label class="d-block">
+                        Branch Name
+                        {{ branch_form.name }}
+                      </label>
+                      {{ branch_form.name.errors }}
+                    </div>
+                    <div class="col-8">
+                      <label class="d-block">
+                        Description
+                        {{ branch_form.description }}
+                      </label>
+                      {{ branch_form.description.errors }}
+                    </div>
+                    <div class="col-1">
+                      <label class="d-block">
+                        Ratio
+                        {% if form.instance.equal_branch_ratio %}
+                          <p>Equal</p>
+                        {% else %}
+                          {{ branch_form.ratio }}
+                        {% endif %}
+                      </label>
+                      {{ branch_form.ratio.errors }}
+                    </div>
+                  </div>
+                </div>
+                {% if not forloop.first %}
+                  <a class="text-primary position-absolute top-0 end-0 m-2"
+                     hx-post="{% url 'nimbus-new-delete-branch' slug=experiment.slug %}"
+                     hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'
+                     hx-params="branch_id"
+                     hx-vals='{"branch_id": {{ branch_form.instance.id }} }'
+                     hx-select="#branches"
+                     hx-target="#branches">
```

Comment:
We should consider adding a templatetag that generates all these htmx attributes based on Python objects instead of having to stringify them in the HTML.

Reply:
Ya that's a good idea.  I'm hoping we do a big ol pass on this after we get the basics in place and figure out ways to make the HTMX patterns more consistent and reusable, so I'd say we can find patterns to extract after we get all the basic mechanics in place.